lib/Net/SIP/Simple/Call.pm

```diff
@@ -452,9 +452,17 @@ sub dtmf {
 	    $type = $self->{param}{sdp_peer}
 		&& $self->{param}{sdp_peer}->name2int('telephone-event/8000','audio');
 	} elsif ( $m eq 'audio' ) {
-	    $type = $self->{param}{sdp_peer}
-		&& $self->{param}{sdp_peer}->name2int('PCMU/8000','audio')
-		|| 0; # default id for PCMU/8000
+	    my $sdp_peer = $self->{param}{sdp_peer};
+	    if ($sdp_peer) {
+		$type = $sdp_peer->name2int('PCMU/8000','audio') unless defined $type;
+		$type = $sdp_peer->name2int('PCMA/8000','audio') unless defined $type;
```

Comment:
This line will never be executed since $type is already defined in the previous line.

Reply:
This is not truth. `name2int` may return undef if `PCMU/8000` on previous line does not exists in `lines` part of sdp peer. This happens when SDP contains only fmt id (0) without fmt name (PCMU).

Reply:
I changed code to use `//`.